src/main/java/uk/gov/justice/digital/service/ValidationService.java

```diff
@@ -39,18 +38,13 @@ public ValidationService(ViolationService violationService) {
     }
 
     public Dataset<Row> handleValidation(SparkSession spark, Dataset<Row> dataFrame, SourceReference sourceReference, StructType inferredSchema, ViolationService.ZoneName zoneName) {
-        try {
-            val maybeValidRows = validateRows(dataFrame, sourceReference, inferredSchema);
-            val validRows = maybeValidRows.filter(col(ERROR).isNull()).drop(ERROR);
-            val invalidRows = maybeValidRows.filter(col(ERROR).isNotNull());
-            if(!invalidRows.isEmpty()) {
-                violationService.handleViolation(spark, invalidRows, sourceReference.getSource(), sourceReference.getTable(), zoneName);
-            }
-            return validRows;
-        } catch (DataStorageException e) {
```

Comment:
DataStorageException is now a RuntimeException anyway and so this preserves the old behaviour